ui/src/utils/mittBus.ts

```diff
@@ -48,6 +48,7 @@ interface Event {
   'open-search': void;
   'file-manager-expired': void;
   'connect-error': void;
+  'close-drawer': void;
 }
 
 // @ts-expect-error mittBus is not typed
```

Comment:
It looks like you provided incomplete code diffs between two versions of an event bus implementation. Please ensure full information about the changes that need to be evaluated before suggesting potential solutions or optimizing improvements.